docs/_docs/windows.md

```diff
@@ -4,7 +4,7 @@ permalink: /docs/windows/
 ---
 
 While Windows is not an officially-supported platform, it can be used to run
-Jekyll with the proper tweaks. If you are using Windows 10 Anniversary Update, 
+Jekyll with the proper tweaks. If you are using Windows 10 Anniversary Update (1607) or newer, 
```

Comment:
When I used the previous Win10 AU (1607), it used the well-known `cmd`. Now Win10 CU (1703) uses `powershell` instead. And when I was installing *as Administrator*, I had found myself to be in the system directory during my performing of those steps.
Again, just mentioning *AU* only makes a future problem for users to understand that it is at all. So my collegues are not smart to know about the number of releases of Win10 for the moment.